apis/Google.Cloud.Diagnostics.AspNetCore3/docs/index.md

```diff
@@ -1,11 +1,9 @@
 {{title}}
 
-`Google.Cloud.Diagnostics.AspNetCore` is an ASP.NET Core instrumentation library for Google Stackdriver.
-It allows for simple integration of Stackdriver into ASP.NET applications with minimal code changes.
+`Google.Cloud.Diagnostics.AspNetCore3` is an ASP.NET Core instrumentation library for Google Logging, Error Reporting and Tracing.
```

Comment:
Whoops, think I've left a bunch of comments before in an earlier commit...

Reply:
Yep, no worries, I went through those as well in case there was something there I'd missed.